src/main/java/org/example/assigment1/ExceptionCheck/Validation.java

```diff
@@ -0,0 +1,64 @@
+package org.example.assigment1.ExceptionCheck;
+
+
+import java.util.*;
+public class Validation extends Exception{
+
+
+    public static boolean validateUserInputForBadInput(String valueToValidate){
+        boolean check=true;
+        try {
+
+            for (int i=0;i < valueToValidate.length();i++) {
+                if (!(valueToValidate.charAt(i)>='0' && valueToValidate.charAt(i)<='9'))
+                    check=false;
+                if (check == false) {
+
+                    throw new BadDataEnteredException("Number of goods Must be in Digits");
+
+                }
+
+            }
+        } catch (BadDataEnteredException x) {
+            System.out.println(x.toString());
+            check =false;
+        }
+        return check;
+    }
+
+        public static boolean validateForName(String nameOfitem){
+        boolean check = true;
+        try{
+            if (nameOfitem.charAt(0) >= '0' && nameOfitem.charAt(0) <= '9'){
```

Comment:
Is this a requirement in the Assignment?

Reply:
It is checking whether the given name can be considered as name or not.